### PR TITLE
Enhancements

### PR DESCRIPTION
**Pull request by Souravgoswami**

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -15,13 +15,40 @@ require("channels")
...
+
+    if(top > 100) {
+      nav.classList.add("white__nav", "box__shadow")
+    } else {
+      nav.classList.remove("white__nav", "box__shadow")
+    }
   }
-};
+
+  function observeComponenets(direction) {
```
Comment (archetype2142) [inline app/javascript/packs/application.js]: `observeComponenets` do u mean `observeComponents`?

File: app/views/layouts/_meta_tags.slim [hunk 2]
```diff
@@ -0,0 +1,14 @@
+meta charset="utf-8"
+meta name="viewport" content="width=device-width,initial-scale=1"
+meta name="description" content="Jalappeno"
+meta name="author" content="Jalappeno"
+meta name="keywords" lang="en-GB" content="Let's talk about your new business ideas"
```
Comment (archetype2142) [inline app/views/layouts/_meta_tags.slim]: can you add content for meta tags from translations? eg. `t('metatags.keywords.content')`

**Review by archetype2142 — CHANGES_REQUESTED** [review]
Loos good besides small changes, feel free to merge after fixing them

File: app/javascript/packs/application.js (hunk 1 above)
Comment (Souravgoswami) [reply app/javascript/packs/application.js]: Oops yes, I will fix this

**Review by archetype2142 — APPROVED** [review]

**Pull request merged**